nameko_amqp_retry/messaging.py

```diff
@@ -0,0 +1,36 @@
+import sys
+
+from nameko.messaging import Consumer as NamekoConsumer
+from nameko_amqp_retry import Backoff, BackoffPublisher
+
+
+class Consumer(NamekoConsumer):
+
+    backoff_publisher = BackoffPublisher()
+
+    def handle_result(self, message, worker_ctx, result=None, exc_info=None):
+
+        if exc_info is not None:
+            exc_type = exc_info[0]
+            if issubclass(exc_type, Backoff):
+
+                # add call stack and modify the current entry to show backoff
+                message.headers['nameko.call_id_stack'] = (
```

Comment:
Looks like the same key is defined as `CALL_ID_STACK_HEADER_KEY` in `nameko_amqp_retry/rpc.py` maybe worth putting it in `constants.py` and share.